src/main/java/mate/academy/service/GenericService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.service;
+
+import java.util.Optional;
+
+public interface GenericService<T> {
```

Comment:
with current flow, I don't see point in generic service, you have here 2 methods but they are duplicated in MovieService